contour.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: contour-1.27
+  version: 1.27.4
+  epoch: 0
+  description: Contour is a Kubernetes ingress controller using Envoy proxy.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - contour=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "0"
+    GO111MODULE: "on"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/projectcontour/contour
+      tag: v${{package.version}}
+      expected-commit: 64dc935bffbfe30ba42b409362770d142d8a4948
+
+  - uses: go/build
+    with:
+      packages: ./cmd/contour
+      output: contour
+      ldflags: |
+        -s -w
+        -X github.com/projectcontour/contour/internal/build.Version=${{package.version}}
+        -X github.com/projectcontour/contour/internal/build.Sha=$(git rev-parse HEAD)
+        -X github.com/projectcontour/contour/internal/build.Branch=$(git rev-parse --abbrev-ref HEAD)
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: projectcontour/contour
+    strip-prefix: v
+
+# requires k8s and kwok alone won't help so going with a simple test in melange
```

Comment:
nit - but i'd personally just drop this comment, otherwise we'd be adding it for most melange files

Reply:
true lol, I will remove it, thanks Mark for the review